src/AvtaleSide/steg/BeregningTilskudd/visningTilskuddsperioder/VisningTilskuddsperioderTabell.tsx

```diff
@@ -24,55 +24,68 @@ const VisningTilskuddsperioderTabell: React.FC<Properties> = ({ className }: Pro
-                        <div
-                            key={index}
-                            className={cls.element('tabell-innslag')}
-                            style={{
-                                borderTop: nyProsent ? '2px solid gray' : 'undefined',
-                            }}
-                        >
-                            <BodyShort size="small">{formaterPeriode(periode.startDato, periode.sluttDato)}</BodyShort>
-                            {innloggetBruker.erNavAnsatt && (
-                                <BodyShort>
-                                    <EtikettStatus tilskuddsperiodestatus={periode.status} size="small" />
-                                </BodyShort>
-                            )}
-                            <BodyShort size="small">
-                                {avtale.tiltakstype === 'VARIG_LONNSTILSKUDD' &&
-                                    periode.status !== 'BEHANDLET_I_ARENA' && <>{periode.lonnstilskuddProsent}%</>}
-                                {(avtale.tiltakstype === 'MIDLERTIDIG_LONNSTILSKUDD' ||
-                                    avtale.tiltakstype === 'SOMMERJOBB') && <>{periode.lonnstilskuddProsent}%</>}
-                            </BodyShort>
-                            <BodyShort size="small" style={{ minWidth: '4rem' }}>
-                                {formaterPenger(periode.beløp)}
-                            </BodyShort>
-                        </div>
-                    );
-                })}
+        <>
+            <Table size={'medium'}>
+                <Table.Header>
+                    <Table.Row>
+                        <Table.HeaderCell textSize={'small'}>Tilskudd for perioder</Table.HeaderCell>
+                        {innloggetBruker.erNavAnsatt && <Table.HeaderCell textSize={'small'}>Status</Table.HeaderCell>}
+                        <Table.HeaderCell textSize={'small'}>Tilskuddsprosent</Table.HeaderCell>
+                        <Table.HeaderCell textSize={'small'}>Inntil</Table.HeaderCell>
+                    </Table.Row>
+                </Table.Header>
+                <Table.Body>
+                    {avtale.tilskuddPeriode
+                        .filter((p: TilskuddsPeriode) => p.aktiv)
+                        .map((periode: TilskuddsPeriode, index: number) => {
+                            const nyProsent: boolean =
+                                index > 0
+                                    ? avtale.tilskuddPeriode[index - 1].lonnstilskuddProsent !==
+                                      periode.lonnstilskuddProsent
+                                    : false;
+                            if (index < startIndexVisning || index > sluttIndexVisning) {
+                                return null;
+                            } else if (index !== 0 && (index === startIndexVisning || index === sluttIndexVisning)) {
+                                return (
+                                    <Table.Row key={index}>
```

Comment:
Det er en anti-pattern å bruke index på et array som en key. Tilskuddsperioder har vel en unik id som heller kan brukes enn index?

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md#disallow-usage-of-array-index-in-keys-reactno-array-index-key